incoming-listener-template.yaml

```diff
@@ -2,6 +2,11 @@ kind: Template
 apiVersion: v1
 metadata:
   name: aiops-incoming-listener
+parameters:
+  - name: KAFKA_SERVER
+    value: platform-mq-cluster-kafka.platform-mq.svc:9092
```

Comment:
The correct value is specified here, so we should be OK for deployment#1, I think.